### PR TITLE
Marketing button dynamic headertag

### PR DESCRIPTION
**Pull request by priteshbsf**
### Description
Fixed Dynamic header tag issue in the title

### Types of changes
- It Introduces you to change the Title of Marketing button with Header Tag (e.g. H1, H2, H3, H4, H5, H6, span, p )

### How has this been tested?
- Checked whether it's working correctly to the frontend and backend

### Checklist:
- [X] My code is tested
- [X] My code follows accessibility standards 
- [X] My code has proper inline documentation 
- [X] I've included any necessary tests 
- [X] I've included developer documentation 
- [X] I've added proper labels to this pull request

### CODE REVIEW

File: config/webpack.config.dev.js [hunk 1]
```diff
@@ -56,8 +56,8 @@ fs.readdir( paths.pluginSrc + "/blocks", function(err, items) {
 		}, function( error, result ) {
 
 			let file_path = result.stats.entry
-			let new_path = file_path.replace( paths.pluginSrc + "/blocks/", "" );
-			new_path = new_path.replace( "/style.scss", "" );
+			let new_path = file_path.replace( paths.pluginSrc + "\\blocks\\", "" );
```
Comment (vrundakansara) [inline config/webpack.config.dev.js:59]: Is this change necessary ?

File: config/webpack.config.prod.js [hunk 2]
```diff
@@ -61,9 +61,9 @@ fs.readdir(paths.pluginSrc + "/blocks", function(err, items) {
 
 			let file_path = result.stats.entry
 
-			let new_path = file_path.replace( paths.pluginSrc + "/blocks/", "" );
+			let new_path = file_path.replace( paths.pluginSrc + "\\blocks\\", "" );
```
Comment (vrundakansara) [inline config/webpack.config.prod.js:64]: Is this change necessary ?

File: config/webpack.config.dev.js (hunk 1 above)
Comment (priteshbsf) [reply config/webpack.config.dev.js:59]: Sorry I forget to Undo this change

File: src/blocks/marketing-button/edit.js [hunk 3]
```diff
@@ -636,6 +657,9 @@ class UAGBMarketingButtonEdit extends Component {
 							</Fragment>
 						) }
 					</PanelBody>
+
+					{/* Typography editing section */}
```
Comment (vrundakansara) [inline src/blocks/marketing-button/edit.js:661]: Why leave commented code?

File: src/blocks/marketing-button/edit.js [hunk 4]
```diff
@@ -267,6 +270,23 @@ class UAGBMarketingButtonEdit extends Component {
 						/>
 					</PanelBody>
 					<PanelBody	title={ __( "Content" ) } initialOpen={ false }	>
+						<h2>{ __( "Title" ) }</h2>
```
Comment (vrundakansara) [inline src/blocks/marketing-button/edit.js:273]: The Title is twice.

File: src/blocks/marketing-button/edit.js (hunk 3 above)
Comment (priteshbsf) [reply src/blocks/marketing-button/edit.js]: Fixed

File: src/blocks/marketing-button/edit.js (hunk 4 above)
Comment (priteshbsf) [reply src/blocks/marketing-button/edit.js]: Fixed
